mm/2s2h/Enhancements/DifficultyOptions/GibdoTradeSequenceOptions.cpp

```diff
@@ -0,0 +1,72 @@
+#include <libultraship/bridge.h>
+#include "2s2h/GameInteractor/GameInteractor.h"
+#include "2s2h/Enhancements/Enhancements.h"
+#include "2s2h/ShipInit.hpp"
+#include <stdarg.h>
+
+#define CVAR_NAME "gEnhancements.Cheats.GibdoTradeSequence"
+#define CVAR CVarGetInteger(CVAR_NAME, GIBDO_TRADE_SEQUENCE_VANILLA)
+
+extern "C" {
+#include "functions.h"
+#include "variables.h"
+#include "overlays/actors/ovl_En_Talk_Gibud/z_en_talk_gibud.h"
+
+// redefinition
+typedef struct {
+    /* 0x0 */ PlayerItemAction itemAction;
+    /* 0x4 */ ItemId item;
+    /* 0x8 */ s32 amount;
+    /* 0xC */ s16 isBottledItem;
+} EnTalkGibudRequestedItem; // size = 0x10
+}
+
+static EnTalkGibudRequestedItem redPotionRequestedItem = { PLAYER_IA_BOTTLE_POTION_RED, ITEM_POTION_RED, 1, true };
+
+void RegisterGibdoTradeSequenceOptions() {
+    COND_VB_SHOULD(VB_GIBDO_TRADE_SEQUENCE_SUFFICIENT_QUANTITY_PRESENTED, CVAR != GIBDO_TRADE_SEQUENCE_VANILLA, {
+        ItemId requestedItemId = va_arg(args, ItemId);
```

Comment:
Similar to the other suggestions for enums passed through va_args:
```suggestion
        ItemId requestedItemId = (ItemId)va_arg(args, int);
```